client/src/hooks/useStakeholders/useStakeholders.js

```diff
@@ -47,77 +49,94 @@ export function useStakeholders() {
     }
   };
 
-  useEffect(() => {
-    const fetchCategories = async (latitude = null, longitude = null) => {
-      const {
-        FETCH_FAILURE,
-        FETCH_REQUEST,
-        FETCH_SUCCESS
-      } = actionTypes.CATEGORIES;
-
-      const {
-        searchString,
-        selectedLatitude,
-        selectedLongitude,
-        selectedDistance
-      } = initialState;
+  const fetchCategories = async () => {
+    const {
+      FETCH_FAILURE,
+      FETCH_REQUEST,
+      FETCH_SUCCESS
+    } = actionTypes.CATEGORIES;
 
-      dispatch({ type: FETCH_REQUEST });
-      try {
-        const allCategories = await categoryService.getAll();
-        const categories = allCategories.filter(category => !category.inactive);
+    dispatch({ type: FETCH_REQUEST });
+    try {
+      const allCategories = await categoryService.getAll();
+      const categories = allCategories.filter(category => !category.inactive);
 
-        const selectedCategories = categories.filter(
-          category =>
-            category.id === 1 || category.id === 8 || category.id === 9
-        ); // setting the initial selection to FoodPantry, Food Bank, Soup Kitchen
-        dispatch({ type: FETCH_SUCCESS, categories, selectedCategories });
+      const selectedCategories = categories.filter(
+        category => category.id === 1 || category.id === 8 || category.id === 9
+      ); // setting the initial selection to FoodPantry, Food Bank, Soup Kitchen
+      dispatch({ type: FETCH_SUCCESS, categories, selectedCategories });
+    } catch (error) {
+      dispatch({ type: FETCH_FAILURE, error });
+    }
+  };
 
-        if (latitude && longitude) {
-          search(
-            searchString,
-            latitude,
-            longitude,
-            selectedCategories,
-            selectedDistance
-          );
-        } else {
-          search(
-            searchString,
-            selectedLatitude,
-            selectedLongitude,
-            selectedCategories,
-            selectedDistance
-          );
-        }
-      } catch (error) {
-        dispatch({ type: FETCH_FAILURE, error });
-      }
-    };
+  const fetchLocation = async () => {
```

Comment:
After looking through what you're doing, I took a stab at simplifying this function a little bit, since we don't really need it to be an async function anymore. We're not waiting on anything really in order to fire off that initial `search(...initialStateParams)` so here's what I ended up with:
```
  const fetchLocation = () => {
    const {
      FETCH_FAILURE,
      FETCH_REQUEST,
      FETCH_SUCCESS,
    } = actionTypes.LOCATION;

    dispatch({ type: FETCH_REQUEST });

    if (navigator.geolocation) {
      navigator.geolocation.getCurrentPosition(
        position => {
          if (!position) {
            dispatch({
              type: FETCH_SUCCESS,
              userCoordinates: { latitude: null, longitude: null },
            });
          }

          const userCoordinates = {
            latitude: position.coords.latitude,
            longitude: position.coords.longitude,
          };
          dispatch({ type: FETCH_SUCCESS, userCoordinates });
        },
        error => {
          dispatch({ type: FETCH_FAILURE, error });
        },
      );
    } else {
      // If browser location permission is denied, the request is
      // "successful", but the result is null coordinates.
      dispatch({
        type: FETCH_SUCCESS,
        userCoordinates: { latitude: null, longitude: null },
      });
    }
  };

```